metametamodel/metametamodel.adoc

```diff
@@ -577,7 +577,7 @@ The language itself has key `LIonCore-M3`.{fn-org91}
 |<<Reference>> |Concept |Reference
 |===
 
-==== Ids of Built-in elements
+==== Ids of built-in elements
```

Comment:
It seems that in other files you moved to capitalized titles. Does it depend on the level of the header?

Reply:
I tried to normalize to
* Document titles are all capitalized
* Headers are sentence capitalized